blackjack.js

```diff
@@ -42,9 +114,31 @@ const dealerShouldDraw = (dealerHand) => {
  */
 const determineWinner = (playerScore, dealerScore) => {
```

Comment:
unable to test without the upstream code fixed, but this looks on the right track.